app/models/recipy.rb

```diff
@@ -0,0 +1,3 @@
+class Recipy < ApplicationRecord
+  belongs_to :user
+end
```

Comment:
@mahbubul14 we  forgot about validations I added mine in the next PR make sure next time to add validation to your recipes model